lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -700,7 +713,9 @@ export abstract class CacheManager implements ICacheManager {
      * @param scopes
      * @param authScheme
      */
-    readAccessTokenFromCache(clientId: string, account: AccountInfo, scopes: ScopeSet, authScheme: AuthenticationScheme, keyId?: string): AccessTokenEntity | null {
+    readAccessTokenFromCache(clientId: string, account: AccountInfo, request: BaseAuthRequest): AccessTokenEntity | null {
```

Comment:
Same here

Reply:
What about idTokens?

Reply:
The unified cache schema spec doesn't cover requested claims on the ID token. With these changes, the ID token will match all requests regardless of the requested claims. Should we have a discussion on if we want to have different behavior?

Reply:
Today all claims requests result in a network request to get new access/id tokens. If we're changing the behavior to use cache we need to make sure we don't ignore id tokens otherwise we may break existing scenarios

Reply:
As discussed offline, pending a more in-depth design discussion, we're leaving this as an access token-only behavior, adhering to the cross-platform cache spec given that we'd need to look into the claims object to distinguish between access token and id token claims requests.